### PR TITLE
guillaume didnt even notice my easter egg

### PR DESCRIPTION
**Pull request by antoinerichez**

### CODE REVIEW

File: src/BikeDataProject.Registrations.API/Controllers/WebRegistrationController.cs [hunk 1]
```diff
@@ -79,7 +79,7 @@ public async Task<IActionResult> RegisterStrava(String code)
                     IsHistoryFetched = false
                 };
 
-                if (this._dbContext.Users.FirstOrDefault(u => (u.ProviderUser == user.ProviderUser) || (u.Provider == Constants.WebStravaUserAgent)) == null)
```
Comment (MeurillonGuillaume) [inline src/BikeDataProject.Registrations.API/Controllers/WebRegistrationController.cs:82]: || bad

File: src/BikeDataProject.Registrations.API/Controllers/WebRegistrationController.cs [hunk 2]
```diff
@@ -79,7 +79,7 @@ public async Task<IActionResult> RegisterStrava(String code)
...
+                if (this._dbContext.Users.FirstOrDefault(u => (u.ProviderUser == user.ProviderUser) && (u.Provider == Constants.WebStravaUserAgent)) == null)
```
Comment (MeurillonGuillaume) [inline src/BikeDataProject.Registrations.API/Controllers/WebRegistrationController.cs:82]: && guud

**Review by BertSchoovaerts — APPROVED** [review]

**Review by MeurillonGuillaume — APPROVED** [review]

File: src/BikeDataProject.Registrations.API/Controllers/WebRegistrationController.cs (hunk 1 above)
Comment (antoinerichez) [reply src/BikeDataProject.Registrations.API/Controllers/WebRegistrationController.cs:82]: You didn't even notice it
AHAH GET PRANKED

File: src/BikeDataProject.Registrations.API/Controllers/WebRegistrationController.cs (hunk 2 above)
Comment (antoinerichez) [reply src/BikeDataProject.Registrations.API/Controllers/WebRegistrationController.cs:82]: ZUCCKED

**Pull request merged**